src/bin/tree-sitter-graph/main.rs

```diff
@@ -116,8 +123,9 @@ fn main() -> Result<()> {
         .with_context(|| format!("Cannot execute TSG file {}", tsg_path.display()))?;
 
     let json = matches.is_present("json");
+    let output_path = matches.value_of("output").map(|str| Path::new(str));
     if json {
-        graph.display_json();
+        graph.display_json(output_path);
```

Comment:
Stick an `unwrap_or(())` here to quiesce the warning.

Reply:
Okay, I see it in CI now. I have no idea why this one isn't showing up locally but other warnings are O_o